js/ssai_pulse.js

```diff
@@ -416,14 +423,27 @@ OO.Ads.manager(function(_, $)
      */
     this.onVideoTagFound = function(eventName, videoId, tagType, metadata)
     {
-      OO.log("TAG FOUND w/ args: ", arguments);
-      if(!amc.isLiveStream && !currentOffset)
+      if (!amc.isLiveStream && !currentOffset)
       {
         return null;
       }
+
       var currentId3Object = _parseId3Object(metadata);
       if (currentId3Object)
       {
+
+        if (currentId3Object["time"] < 100) {
```

Comment:
What is 100? 100 seconds, 100ms?  Can you make it a constant otherwise it kind of looks like a magic number. And you reuse it in the next if.

Reply:
@pilievOoyala fixed